lib/travis/sso/login.html

```diff
@@ -19,6 +19,15 @@
       window.setTimeout(function() {
         document.getElementById('warning').style.display = 'inherit';
       }, 10000);
+      function getHostName(url) {
+        var match = url.match(/:\/\/(www[0-9]?\.)?(.[^/:]+)/i);
+        if (match != null && match.length > 2 && typeof match[2] === 'string' && match[2].length > 0) {
+        return match[2];
+        }
+        else {
+            return null;
+        }
+      }
```

Comment:
I don't like this regexp solution - it's hard to understand, hard to debug and hard to predict. I'd like to suggest a little bit simpler approach:
```suggestion
      function getHostName(url = '') {
        return url.split("//")[1].split("/")[0].split("?")[0];
      }
```
What do you think?

Reply:
Perfect 👍